service/service_ha_test.go

```diff
@@ -0,0 +1,295 @@
+package service_test
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/pivotal-cf/cf-redis-smoke-tests/redis"
+	"github.com/pivotal-cf/cf-redis-smoke-tests/service/reporter"
+
+	smokeTestCF "github.com/pivotal-cf/cf-redis-smoke-tests/cf"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/pivotal-cf/cf-redis-smoke-tests/service/utils"
+)
+
+var _ = Describe("Redis HA", Label("ha"), func() {
```

Comment:
Because we are using labels, do we still need to use the "skip" variable?

Reply:
Yes, we have to use that skip variable, that skip variable serves different purpose.

We have a util function for cf cli, which sets skip to true, when certain error has occurred (service quota limit reached). And if skip is set to true we can skip running remaining assertion and tests.

please refer -> https://github.com/pivotal-cf/cf-redis-smoke-tests/blob/f94f2eb727e2011ec4c32bbc2a4eea2cd2564ae7/cf/cf.go#L451